lib/server.js

```diff
@@ -28,6 +28,7 @@ module.exports = (directory, format, width, height, framerate, horizontalFlip, v
 
   // Set up camera stream conversion
   let conversionStream = ffmpeg(cameraStream.stdout)
+    .inputOptions("-framerate", framerate)
```

Comment:
To my knowledge, FFmpeg automatically determines framerate from the input.

Reply:
I believe it only does so with certain video formats. In my experience with raspi-live, the framerate will default to 25fps without this change.

Reply:
I'm seeing the same issue.. without this line, I'm stuck at 25fps. With this line in, it works.
Any blockers from merging this?